test/cpp/api/functional.cpp

```diff
@@ -2858,3 +2858,16 @@ TEST_F(FunctionalTest, BCEWithLogitsLoss) {
     ASSERT_TRUE(torch::isfinite(out2).all().item<bool>());
   }
 }
+
+TEST_F(FunctionalTest, linalg_svd) {
+    // NOTE: this is only a partial test: it tests that when we pass
+    // compute_uv=False, the returned U and VT are empty tensors. We need to
+    // write a C++ test because in Python it has a slightly different behavior
+    // and it returns (None, S, None) instead. The full logic for svd is
```

Comment:
This comment is no longer correct -- is this test still necessary?

Reply:
good catch. The test is no longer necessary since the python and c++ versions are completely equivalent now. Killed by 4b4076dcea